### PR TITLE
Process

### PR DESCRIPTION
**Pull request by adutfoy**
Added documentation for various process-related classes

### CODE REVIEW

**Comment by jschueller** [comment]
+1 very good docstrings !

File: python/src/CompositeProcess_doc.i.in [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+with :math:`t': \cD \rightarrow \cD'` and :math:`v': \cD \times \Rset^d \rightarrow \Rset^q`.
+
+The process :math:`Y: \Omega \times \cD' \rightarrow \Rset^q` is defined on the domain :math:`\cD'` associated to the mesh :math:`\cM'`.
+
+Examples
+--------
+Create the process X:
+
+>>> amplitude = [1.0, 1.0]
```
Comment (jschueller) [inline python/src/CompositeProcess_doc.i.in]: missing openturns import

File: python/src/CompositeProcess_doc.i.in [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+>>> scale = [0.2, 0.3]
+>>> myCovModel = ot.ExponentialModel(2,amplitude,scale)
+>>> myMesh = ot.IntervalMesher([100]*2).build(ot.Interval([0.0]*2, [1.0]*2))
+>>> myXProcess = ot.TemporalNormalProcess(myCovModel, myMesh)
+
+Create a spatial  dynamical function :math:`g_{dyn}` associated to :math:`g: \Rset^2 \mapsto \Rset^2` where :math:`g(x_1,x_2)= (x_1^2, x_1+x_2)`:
+
+>>> import openturns as ot
```
Comment (jschueller) [inline python/src/CompositeProcess_doc.i.in]: instead of here

**Comment by dbarbier** [comment]
+1

**Pull request merged**